.circleci/config.yml

```diff
@@ -84,24 +84,23 @@ jobs:
       - ios/install-dependencies:
             bundle-install: true
             pod-install: true
-      - run: 
+      - run:
           name: Copy Secrets
           command: bundle exec fastlane run configure_apply
       # Start: Swift Package Manager Workaround
       # Ref. https://support.circleci.com/hc/en-us/articles/360044709573-Swift-Package-Manager-fails-to-clone-from-private-Git-repositories
       - run: rm ~/.ssh/id_rsa
       - run: for ip in $(dig @8.8.8.8 github.com +short); do ssh-keyscan github.com,$ip; ssh-keyscan $ip; done 2>/dev/null >> ~/.ssh/known_hosts || true
       # END: Swift Package Manager Workaround
-      - run: 
+      - run:
           name: Install other tools
           command: |
-            brew unlink python@2
```

Comment:
The "Release Build" job for the `4.24.0.0` tag [failed](https://app.circleci.com/pipelines/github/Automattic/simplenote-ios/2136/workflows/7c4e26c3-04d3-4580-80d9-b2dbd257f5c8/jobs/2131) with

`Error: No such keg: /usr/local/Cellar/python@2`

Because there's no keg anymore, we don't need to unlink it.

Reply:
CircleCI must be doing some weird stuff these days. It's surprising how, despite having the environment pinned, we get this change in what's on the machine.

https://github.com/Automattic/simplenote-ios/blob/df7b52b1922f63e82af0593ffa1bd1e5cbf6774b/.circleci/config.yml#L76-L79